.github/workflows/update_copyright.yml

```diff
@@ -6,22 +6,19 @@ on:
 
 jobs:
   update-copyright:
+    if: github.repository_owner == 'cylc'
```

Comment:
I think it might be possible to stop the whole workflow running with this:

```
on:
    schedule:
        branches:
            - 'cylc/**'
```

Reply:
Unfortunately that isn't valid. Only `push` and `pull_request` have the `branches` option https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

Reply:
Dammit.